### PR TITLE
adding Frisby v0.8.5 typings

### PR DESCRIPTION
**Pull request by johnny4753**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [ ] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/frisby/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        toMatchOrBeEmpty(expected: any): boolean;
+        toBeType(expected: any): boolean;
+        toBeTypeOrNull(expected: any): boolean;
+        toBeTypeOrUndefined(expected: any): boolean;
+        toContainJson(expected: any, isNot?: boolean): boolean;
+        toContainJsonTypes(expected: any, isNot?: boolean): boolean;
+    }
+}
+
+declare module "frisby" {
```
Comment (sandersn) [inline types/frisby/index.d.ts]: Don't declare module "frisby", just put the `const` and the `export =` in the top-level.

File: types/frisby/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    after(callback: (...args: any[]) => void): Frisby;
+    afterJSON(callback: (...args: any[]) => void): Frisby;
+    exceptionHandler(fn: (...args: any[]) => void): Frisby;
+    toss(): void;
+
+    create(msg: string): Frisby;
+
+}
+
+declare namespace jasmine {
```
Comment (sandersn) [inline types/frisby/index.d.ts]: change to `declare global { declare namespace jasmine { ...`

File: types/frisby/index.d.ts (hunk 1 above)
Comment (sandersn) [inline types/frisby/index.d.ts]: Ideally, you would even get rid of both lines and the `Frisby` interface, and just turn all its methods into top-level `export function globalSetup(opts: any): any; ...`. But that's a pretty big change so I will merge either way.

**Review by sandersn — CHANGES_REQUESTED** [review]

**Comment by johnny4753** [comment]
Thanks for your code review.
this is my first pull request and I am very honored to contribute to this awesome language.

**Pull request merged**